week-1/1-exercises/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,25 @@
-var mentor1 = "Daniel";
-var mentor2 = "Irina";
-var mentor3 = "Mimi";
-var mentor4 = "Rob";
-var mentor5 = "Yohannes";
+//Exercise 1//
+function percentProgram(firstNumber, secondNumber) {
+  let overallTotal = firstNumber + secondNumber;
```

Comment:
Here you have demonstrated using function parameters nicely so I'm sure you can correct the other exercises above :)